bin/schemas/story.json

```diff
@@ -0,0 +1,839 @@
+              "title": "Red",
+              "minimum": 0,
+              "maximum": 255
+            },
+            "g": {
+              "type": "integer",
+              "title": "Green",
+              "minimum": 0,
+              "maximum": 255
+            },
+            "b": {
+              "type": "integer",
+              "title": "Blue",
+              "minimum": 0,
+              "maximum": 255
+            },
+            "a": {
+              "type": "number",
+              "title": "Alpha",
+              "minimum": 0,
+              "maximum": 1
+            }
+          }
+        }
+      }
+    },
+    "linearPattern": {
+      "type": "object",
+      "title": "Linear color pattern",
+      "required": ["type", "stops"],
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["none", "solid", "linear", "radial"]
+        },
+        "stops": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "type": "object",
+            "allOf": [
+              {
+                "$ref": "#/$defs/solidPattern"
+              }
+            ],
+            "properties": {
+              "position": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1
+              }
+            }
+          }
+        },
+        "alpha": {
+          "type": "number",
+          "minimum": 0,
+          "maximum": 1
+        },
+        "rotation": {
+          "type": "number",
+          "minimum": -360,
+          "maximum": 360
+        }
+      }
+    },
+    "pageTemplateType": {
+      "type": "string",
+      "title": "The pageTemplateType Schema",
+      "enum": [
+        "",
+        "cover",
+        "editorial",
+        "list",
+        "quote",
+        "section",
+        "table",
+        "steps"
+      ]
+    },
+    "page": {
+      "type": "object",
+      "title": "Individual Page",
+      "required": ["backgroundColor", "elements", "id", "type"],
+      "properties": {
+        "animations": {
+          "type": "array",
+          "title": "Animations on this page",
+          "items": {
+            "$ref": "#/$defs/animation"
+          }
+        },
+        "elements": {
+          "type": "array",
+          "minItems": 1,
+          "title": "The page's elements",
+          "items": {
+            "anyOf": [
+              { "$ref": "#/$defs/element" },
+              { "$ref": "#/$defs/mediaElement" },
+              { "$ref": "#/$defs/shapeElement" },
+              { "$ref": "#/$defs/stickerElement" }
+            ]
+          }
+        },
+        "backgroundColor": {
+          "type": "object",
+          "title": "Background color",
+          "$ref": "#/$defs/pattern"
+        },
+        "defaultBackgroundElement": {
+          "type": "object",
+          "title": "The default background element",
+          "oneOf": [
+            { "$ref": "#/$defs/mediaElement" },
```

Comment:
<!-- general note -->
**📝 note:**

The default background element can only be a shape, if it's not a shape, it's not the default background element.